mobify/astro-tutorial
# Build js updates

Imports `build-js.sh` fixes from Astro. Makes it aware of `nvm` so we depend on `user-env.sh` much less.

JIRA: N/A
Linked PRs: https://github.com/mobify/astro/pull/554

## Changes
- Make `build-js.sh` aware of NVM so it can find node on its own
- Suppress progress and colours when building in Xcode and Android Studio

## How to test-drive this PR
- Build the app in Xcode
- Build the app in Android Studio
Note that the output isn't messed up with shell escape codes for colours

## TODOS:
~~- [ ] Update tutorial documentation in the `dev` folder (in the Astro repo) to match the modified tutorial.~~
~~- [ ] Preview the docs to make sure your changes look good (see [here](https://github.com/mobify/astro#documentation) for instructions on how to preview).~~
